frontend/src/metabase/dashboard/components/AddSeriesModal/AddSeriesModal.tsx

```diff
@@ -108,27 +142,22 @@ class AddSeriesModal extends Component {
           </div>
           <div className="flex-full ml2 mr1 relative">
             <Visualization
+              canRemoveSeries={CAN_REMOVE_SERIES}
               className="spread"
               rawSeries={series}
               showTitle
               isDashboard
               isMultiseries
               onRemoveSeries={this.handleRemoveSeries}
```

Comment:
Reference to `this` was being lost here, so I had to bind `this.handleRemoveSeries`.